js/components/graph/__tests__/TestGraph.js

```diff
@@ -31,7 +31,7 @@ export default class TestGraph {
     }
 
     const rtlGraph = render(<Graph {...graphProps} />)
-    await waitFor(() => rtlGraph.queryByText("AAA100") !== null)
+    await waitFor(() => rtlGraph.getByText("AAA100") !== null)
```

Comment:
This is great; you probably found in your [research](https://testing-library.com/docs/dom-testing-library/api-async#findby-queries) that "`findBy` methods are a combination of `getBy` [queries](https://testing-library.com/docs/queries/about#types-of-queries) and [`waitFor`](https://testing-library.com/docs/dom-testing-library/api-async#waitfor)". So here, and the other places you fixed similar issues, it seems like you can just use `await findByText` instead?